### DISCUSSION TITLE
Customization does not work

### DISCUSSION THREAD

**Discussion by Anatorusan** [tailwindlabs/tailwindcss#10461] [Help] [answered]
Hi,
I am absolutely new to the tailwind, previously worked only with pure css.

I tried to create custom classes, but no effect followed. Meanwhile, the standard tailwind classes work. I strictly followed the documentation instructions. There are no errors in terminal, it correctly rebuilds after each save.

**tailwind.config.js**

`/** @type {import('tailwindcss').Config} */`
`module.exports = {`
`  content: ["./src/**/*.{html,js}"],`
`  theme: {`
`    extend: {`
`      backgroundColor: {`
`        'altRed': '#8a0303',`
`      },`
`      maxWidth: {`
`        'fullScr': '1440px'`
`      }`
`    },`
`  },`
`  plugins: [],`
`}`

**input.css**

`@tailwind base;`
`@tailwind components;`
`@tailwind utilities;`

Please help, I feel that I am missing something very simple.

Thank you!

**Comment by wongjn** [comment, marked as answer]
Your configuration seems to work fine in [Tailwind Play](https://play.tailwindcss.com/J9smCPrcvh) (open the *Generated CSS* panel). Thus, it seems like it could be some other misconfiguration/misuse in your project. Some things to check:
- The `content` file globs sufficiently cover all files that use Tailwind classes.
- You are not use string interpolation/concatenation to build the class name strings.
- The class name strings corresponding to your configuration have actually been used in scanned code.

Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by Anatorusan** [reply]
I found the issue - I used bare class names, without 'max-w-' or 'bg-' prefixes, which you have used in the Tailwind Play. Tried it and now it works. 

Thank you!
